### DISCUSSION TITLE
Osu CS Glitch

### DISCUSSION THREAD

**Discussion by Drazzens** [ppy/osu#19794] [Q&A] [answered]
I found a glitch where circle size seems to change mid level if you switch your resolution to stretched and then back to normal I was seeing what stretched Osu! would look like but then found this bug it also shows up in replay but not after restarting game.

https://user-images.githubusercontent.com/111329172/184786365-cee51a9f-3fcb-4da0-9aff-1a2d0b9c734f.mp4

https://user-images.githubusercontent.com/111329172/184786537-82d710d1-ee76-4322-a98c-0d036c8fc7dc.mp4

https://user-images.githubusercontent.com/111329172/184786646-48cd6ce3-405b-4744-b74e-05fd5afc048e.mp4

**Comment by peppy** [comment, marked as answer]
This is not lazer. You can report the issue at https://github.com/ppy/osu-stable-issues/issues, but keep in mind that stable is feature locked and we likely won't address issues only affecting a few users.
